src/PKSim.Core/Mappers/PathAndValueBuildingBlockMapper.cs

```diff
@@ -63,7 +63,7 @@ protected TBuildingBlock CreateBaseObject(TPKSimBuildingBlock pkSimBuildingBlock
          return buildingBlock;
       }
 
-      public virtual TBuilder MapParameter(IParameter parameter, TPKSimBuildingBlock pkSimBuildingBlock)
+      protected virtual TBuilder CreateParameter(IParameter parameter, TPKSimBuildingBlock pkSimBuildingBlock)
```

Comment:
Interestingly, the create Parameter (formerly map parameter), was not mapping formula anympore. Not sure why we did this like that but anyways, I am exposing something else now where we actually MAP formulas